app/src/main/java/net/skyscanner/backpack/demo/compose/BpkBottomNavStory.kt

```diff
@@ -38,32 +38,39 @@ import net.skyscanner.backpack.compose.tokens.Trips
 import net.skyscanner.backpack.demo.R
 
 @Composable
-fun BpkBottomNavStory() {
-  Column(
+fun BpkBottomNavStory(modifier: Modifier = Modifier) {
+  Box(
     modifier = Modifier.fillMaxSize(),
-    verticalArrangement = Arrangement.Bottom,
+    contentAlignment = Alignment.BottomEnd,
   ) {
-    BpkBottomNavSample()
+    BpkBottomNavSample(modifier = modifier)
```

Comment:
you'll want to set the modifier on your top level composable, which in your case is `Box`